purchase_custom_search_product/models/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
No necessary here.

Reply:
Yes better to have an empty file if there is no python code in it.

Reply:
Done